asv_ws/src/motion/dp_controller/README.md

```diff
@@ -0,0 +1,41 @@
+## ASV note
+This is a tried and tested node, but is made for full 6DOF control. We will not need to use quaternions, or care about anything other than DOF 1,2 and 6. We can either rewrite this code, or use it as a reference to write a new DP controller entirely.
+
+## DP controller
+
+#### Subscribes to:
+* /odometry/filtered
+
+#### Publishes to:
+* /auv/thruster_manager/input
```

Comment:
this should perhaps be 

/asv/thruster_manager/input
but i expect there to be a lot of these that need to be updated.

Reply:
Definitely! This is also outdated since I picked a previous version of the DP controller that I think will be easier to work with.